platforms/web/test-e2e/tests/language_test.ts

```diff
@@ -58,13 +58,49 @@ Scenario('English language is selected when the locale is `en-GB`', async ({ I }
   await assertActiveLanguage(I, 'en');
 });
 
+Scenario('English title and description are displayed when the locale is `en-GB`', async ({ I }) => {
+  I.restartBrowser({ locale: 'en-GB' });
+  I.useConfig(testConfigs.basicNoAuth);
+  await assertActiveLanguage(I, 'en');
+
+  await I.openVideoCard(constants.agent327Title);
+  I.see(constants.agent327Title);
+  I.see(constants.agent327Description);
+
+  await I.openVideoCard(constants.bigBuckBunnyTitle);
+  I.see(constants.bigBuckBunnyTitle);
+  I.see(constants.bigBuckBunnyDescription);
+
+  await I.openVideoCard(constants.elephantsDreamTitle);
+  I.see(constants.elephantsDreamTitle);
+  I.see(constants.elephantsDreamDescription);
+});
```

Comment:
Here when we use `openVideoCard` function on the movie page, we try to find the item within "related" shelf. It is also possible that the item is not shown there. I would suggest to go back to the home page before opening a new movie page (`I.amOnPage(constants.baseUrl)`)